src/Common/nunit/ConsoleOptions.cs

```diff
@@ -382,8 +382,10 @@ private void ConfigureOptions()
             this.Add("trace=", "Set internal trace {LEVEL}.\nValues: Off, Error, Warning, Info, Verbose (Debug)",
                 v => InternalTraceLevel = RequiredValue(v, "--trace", "Off", "Error", "Warning", "Info", "Verbose", "Debug"));
 
+#if !NETCF
             this.Add("teamcity", "Turns on use of TeamCity service messages.",
                 v => TeamCity = v != null);
+#endif
```

Comment:
In that case, I suppose this one can be left as well. It does no harm and reduces the number of #ifs.